frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/CreateCluster.tsx

```diff
@@ -295,8 +295,14 @@ export default function CreateClusterPage() {
                 })
                 break
             case 'templateName':
-                control.available = curatorTemplates.map((template) => template.metadata.name)
-                setAvailableTemplates(control, curatorTemplates)
```

Comment:
Randy and I determined that this function call was redundant and repeating the same task as in the line above it (line 298). I removed this function call and filtered out templates where no secret exists.